Add Insights prerequisite (#3735)
theforeman/foreman-documentation: guides/common/modules/proc_performing-post-upgrade-tasks.adoc

This migration takes time, so a pre-migration runs automatically after the upgrade to {ProjectVersion} to reduce future upgrade downtime.
While the pre-migration is running, {ProjectServer} is fully functional but uses more hardware resources.
endif::[]
ifdef::satellite[]
ifeval::["{mode}" == "connected"]

> . If you use Red[nbsp]Hat Insights, re-register the {ProjectServer} to Insights.

Reviewer: ```suggestion
. If you use Red{nbsp}Hat Insights, re-register your {ProjectServer} to Insights.
```

blocking: Please use curly brackets for attributes such as "nbsp".

non-blocking: Please use "your Foreman Server" in favor of "the Foreman Server". Applies also to line 16 and 18 in the other file.

> For more information, see https://{RHDocsBaseURL}red_hat_insights/1-latest/html/client_configuration_guide_for_red_hat_insights_with_fedramp/assembly-client-configuring-insights-client#proc-reregistering-system-insights_insights-cg-configuring-client[Re-registering your system with Red Hat Insights].

Reviewer: ```suggestion
For more information, see {RHDocsBaseURL}red_hat_insights/1-latest/html/client_configuration_guide_for_red_hat_insights_with_fedramp/assembly-client-configuring-insights-client#proc-reregistering-system-insights_insights-cg-configuring-client[Re-registering your system with Red Hat Insights].
```

> For more information, see https://docs.redhat.com/en/documentation/red_hat_insights/1-latest/html/client_configuration_guide_for_red_hat_insights_with_fedramp/assembly-client-configuring-insights-client#proc-reregistering-system-insights_insights-cg-configuring-client[Re-registering your system with Red Hat Insights].

Reviewer 2: ```suggestion
For more information, see {RHDocsBaseURL}red_hat_insights/1-latest/html/client_configuration_guide_for_red_hat_insights_with_fedramp/assembly-client-configuring-insights-client#proc-reregistering-system-insights_insights-cg-configuring-client[Re-registering your system with Red Hat Insights].
```
